Makefile

```diff
@@ -56,22 +60,21 @@ dist/apple-touch-icon.png: dist/icon-180x180.png | dist
 	$(INSTALL) -m 0664 $< $@
 dist/favicon.ico: dist/icon-32x32.png | dist
 	$(CONVERT) $^ $@
-dist/main.css: www/main.scss node_modules | dist
-	$(SASSC) --style compressed $< $@
```

Comment:
If you get rid of this, you should also remove the `SASSC` definition at the top of this Makefile